dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -234,25 +235,27 @@ public void filteredTest() throws Exception
             HPCCRecord record = new HPCCRecord(fields, recordDef);
             records.add(record);
         }
-        writeFile(records, "benchmark::large_record_8MB::10rows", recordDef);
+        writeFile(records, "benchmark::large_record_8MB::10rows", recordDef,connTO);
 
         HPCCFile file = new HPCCFile("benchmark::large_record_8MB::10rows", connString , hpccUser, hpccPass);
         file.setFilter("key = 0 OR key > 10");
 
-        records = readFile(file);
+        records = readFile(file, connTO);
         if (records.size() != 1)
         {
             Assert.fail("Failed to read filtered record dataset");
         }
     }
 
-    public List<HPCCRecord> readFile(HPCCFile file) throws Exception
+    public List<HPCCRecord> readFile(HPCCFile file, Integer connectTimeoutMillis) throws Exception
     {
         if (file == null)
         {
             Assert.fail("HPCCFile construction failed.");
         }
-
+        if (connectTimeoutMillis != null) {
+            file.setFileAccessExpirySecs(connectTimeoutMillis/1000);
```

Comment:
typically the connect timeout would be considerably shorter than the fileaccessexpiry, i'm not sure what we get from combining the two values. Albeit, we prob do need a junit test w/ very short/log expiry values...

Reply:
I was getting a file access expiry error (connecting over vpn) until I added this to the test.

Reply:
fair enough, but my issue has to do w/ the reuse of the connTO, we need a dfsclient test specific fileAccessExpiry variable

private final static Integer fileAccessExpiry = System.getProperty("fileAccessExpirySecs")==null?null:Integer.valueOf(System.getProperty("fileAccessExpirySecs"));